markdown/3.x/en/documentation/include-scripts.md

```diff
@@ -24,16 +24,19 @@ Depending on the browser's page loading time you might encounter ugly flashes of
 ```html
 <ff-record-list unresolved></ff-record-list>
 ```
-Also add the following CSS rule to the top of the page before the appearance of FACT-Finder Web Components:
+Also add suitable CSS rule to the top of the page before the appearance of FACT-Finder Web Components.
```

Comment:
```suggestion
Also add suitable CSS rules to the top of the page before FACT-Finder Web Components are referenced.
```
_"the appearance of WebComponents"_ sounds confusing in the context of elements' visibility. I think we should take the opportunity and improve this part. But feel free to tweak it if you can think of something better.